participants/nikolas_odonnell.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Nikolas O'Donnell",
+  "company": "iwantmyname",
+  "tags": ["JavaScript", "React", "Phoenix", "Elixir"],
+  "when": {
+    "friday": true,
+    "friday_party": true,
+    "saturday": true
+  },
+  "email": "nikodonnell@gmail.com",
+  "dietary_requirements": "vegetarian",
+  "what_is_my_connection_to_javascript": "I love JS - I've chatted with Brendan Eich, when I was just starting out. I'm still a new developer, but working full time, most recently picking up React.",
+  "what_can_i_contribute": "I'm super enthusiatic, ask questions, and generally love web development and JavaScript so will be happy to learn as much as I can. Whilst working on side projects and listening to more seasoned professionals.",
+  "tshirt": "/^(W|M)-(M-L)$/",
```

Comment:
Shirt Size needs to follow this regex. Possible values are for example `W-M` or `M-L`. 
The first letter is the shirt-gender-type (W=Women, M=Man), the second the size.